lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -91,8 +91,12 @@ def test_get_receipt_page_url(self):
         self.assertEqual(url, expected_url)
 
     @override_settings(ECOMMERCE_PUBLIC_URL_ROOT='http://ecommerce_url')
-    def test_checkout_page_url(self):
+    @ddt.data(
```

Comment:
Why are these changes necessary for the program about page? This seems like it belongs in a separate commit.